docs/source/federation-spec.md

```diff
@@ -252,6 +252,42 @@ type Product @key(fields: "upc") @key(fields: "sku") {
 
 > Note: Repeated directives (in this case, `@key`, used multiple times) require support by the underlying GraphQL implementation.
 
+### `@link`
+
+```graphql
+directive @link(
+  url: String, 
+  as: String, 
+  for: link__Purpose, 
+  import: [link__Import]
+) repeatable on SCHEMA
+```
+
+The `@link` directive links definitions within the document to external schemas. External schemas are identified by their `url`, which optionally ends with a name and version in the form `name`/v`major`.`minor`.
```

Comment:
I think the deploy preview made it seem a little unclear what the URL should be, I'd recommend splitting this out into two different sentences, like so:

```
External schemas are identified by their `url`, which optionally ends with a name and version with the following format:

`{NAME}`/v`{MAJOR}`.`{MINOR}`
```